assets/airports/lrop.json

```diff
@@ -0,0 +1,715 @@
+              "NETUL": ["NETUL"]
+            },
+            "draw": [
+                ["OP887", "NETUL"],
+                ["OP888", "NETUL"],
+                ["OP662", "OP667", "ETOGO", "NETUL"],
+                ["OP661", "OP667", "ETOGO", "NETUL"]
+            ]
+        },
+        "IDARU": {
+            "icao": "IDARU",
+            "name": "Idaru Departure",
+            "suffix": {"08R": "1K", "08L": "1K", "26R": "1M", "26L": "1M"},
+            "rwy": {
+                "08R": ["OP888"],
+                "08L": ["OP887"],
+                "26R": [["OP662", "S210-"], ["OP667", "S270-"], "ETOGO"],
+                "26L": [["OP661", "S210-"], ["OP667", "S270-"], "ETOGO"]
+            },
+            "body": [],
+            "exitPoints": {
+              "IDARU": ["IDARU"]
+            },
+            "draw": [
+                ["OP882", "OP884", "OP887", "IDARU"],
+                ["OP882", "OP884", "OP888", "IDARU"],
+                ["OP662", "OP667", "ETOGO", "IDARU"],
+                ["OP661", "OP667", "ETOGO", "IDARU"]
+            ]
+        },
+        "DENAK": {
+            "icao": "DENAK",
+            "name": "Denak Departure",
+            "suffix": {"08R": "1K", "08L": "1K", "26R": "1M", "26L": "1M"},
+            "rwy": {
+                "08R": [["OP884", "S270-"], "ETUXA", "EKSUN"],
+                "08L": [["OP883", "S270-"], "ETUXA", "EKSUN"],
+                "26R": [["OP662", "S210-"], "OP667"],
+                "26L": [["OP661", "S210-"], "OP667"]
+            },
+            "body": [],
+            "exitPoints": {
+              "DENAK": [["DENAK", "A110+"]]
+            },
+            "draw": [
+                ["OP884", "ETUXA", "EKSUN", "DENAK"],
+                ["OP883", "ETUXA", "EKSUN", "DENAK"],
+                ["OP662", "OP667", "DENAK"],
+                ["OP661", "OP667", "DENAK"]
+            ]
+        },
+        "BUKEL": {
+            "icao": "BUKEL",
+            "name": "Bukel Departure",
+            "suffix": {"08R": "1K", "08L": "1K", "26R": "1M", "26L": "1M"},
+            "rwy": {
+                "08R": [["OP884", "S270-"], "ETUXA"],
+                "08L": [["OP883", "S270-"], "ETUXA"],
+                "26R": [["OP662", "S210-"], ["OP667", "A45+"], "DESUX"],
+                "26L": [["OP661", "S210-"], ["OP667", "A45+"], "DESUX"]
+            },
+            "body": [],
+            "exitPoints": {
+              "BUKEL": [["BUKEL", "A110+"]]
+            },
+            "draw": [
+                ["OP884", "ETUXA", "BUKEL"],
+                ["OP883", "ETUXA", "BUKEL"],
+                ["OP662", "OP667", "DESUX", "BUKEL"],
+                ["OP661", "OP667", "DESUX", "BUKEL"]
+            ]
+        }
+    },
+    "stars": {
+        "DENAK" : {
+            "icao": "DENAK",
+            "name": "Denak Arrival",
+            "suffix": {"08R": "1U", "08L": "1U", "26R": "2X", "26L": "2X"},
+            "entryPoints": {
+                "DENAK": ["FOCSA", "URELA", "LAPKA" ]
```

Comment:
That's probably because you need to change the spawn pattern too. If you change DENAK here to FOCSA, the spawnPattern that corresponds to this one should be `FOCSA.DENAK.LROP`